### ISSUE TITLE
Poor rendering performance when using several concurrent `<Video>` elements

### ISSUE THREAD

**Issue by skinandbones** [remotion-dev/remotion#993]
When a composition contains many `<Video>` elements within the same frame, rendering performance degrades as more `<Video>`s are added to the frame.

Test rendering a 10s composition on a Mac M1 Pro (16GB)

| Number of videos | Render time |
| --- | --- |
| 1 | 54s |
| 4 | 77s |
| 8 | 92s |
| 12 | 106s |
| 16 | 122s |

Note: increasing render concurrency makes the rendering unreliable due to timeout issues in delay/continue render (separate issue?).   I have not tested these use cases on Lambda which is my intended target.

# Bug Report 🐛

To reproduce the issue, use [this Remotion project](https://github.com/skinandbones/remotion-multiple-videos-repro).

https://github.com/skinandbones/remotion-multiple-videos-repro

```zsh
git clone https://github.com/skinandbones/remotion-multiple-videos-repro.git
cd remotion-multiple-videos-repro
yarn
yarn build
```

Or, checkout the project and run the render substituting one of the following compositions:

* `Videos-1`
* `Videos-4`
* `Videos-8`
* `Videos-12`
* `Videos-16`

```
yarn run remotion render src/index.tsx Videos-16 out/video.mp4 --concurrency=1 --gl=angle
```

**Comment by skinandbones** [comment]
Rendering time can be improved by rendering the videos at a FPS lower than the composition.   For example, if we use `<Freeze>` and only update our video every other frame (effectively 15fps) then the render time improves 2x.   The relationship seems linear.   So, lowering the effective FPS of the videos may be a workaround to slow renders if you can tolerate it.

| Number of videos | Render time |
| --- | --- |
| 16 | 71s |

An example showing how I did this in the reproducer code is here:
https://github.com/skinandbones/remotion-multiple-videos-repro/pull/1

Basically, just this ...

```jsx
<Freeze frame={Math.floor(frame / 2.0) * 2.0}>
    <Video muted src={src} />
</Freeze>
```
